ddtrace/utils/formats.py

```diff
@@ -0,0 +1,58 @@
+import os
+
+
+def get_env(integration, variable, default=None):
+    """Retrieves environment variables value for the given integration. It must be used
+    for consistency between integrations. The implementation is backward compatible
+    with legacy nomenclature:
+        * `DATADOG_` is a legacy prefix with lower priority
+        * `DD_` environment variables have the highest priority
+        * the environment variable is built concatenating `integration` and `variable`
+          arguments
+        * return `default` otherwise
+    """
+    key = '{}_{}'.format(integration, variable).upper()
+    legacy_env = 'DATADOG_{}'.format(key)
+    env = 'DD_{}'.format(key)
+
+    # [Backward compatibility]: `DATADOG_` variables should be supported;
+    # add a deprecation warning later if it's used, so that we can drop the key
```

Comment:
Is it time to do it? or we might forget.

Reply:
oh good call! I totally forgot about it